fleetspeak/src/client/daemonservice/execution/execution_test.go

```diff
@@ -268,9 +261,9 @@ func TestStats(t *testing.T) {
 		t.Error("Last resource-usage report from finished process was not received.")
 	}
 
-	// We expect floor(2800 / 500) regular resource-usage reports, plus the last one sent after
+	// We expect floor(3200 / 1000) regular resource-usage reports, plus the last one sent after
```

Comment:
A resource-usage message will be sent every 2 seconds, but the execution only runs for 3.2 seconds, so we expect 2 messages, not 4.

Reply:
done